package.json

```diff
@@ -28,22 +34,28 @@
   },
   "homepage": "https://github.com/SortableJS/angular-sortablejs#readme",
   "dependencies": {
-    "@angular/common": "^2.0.0",
-    "@angular/core": "^2.0.0",
-    "@angular/forms": "^2.0.0",
-    "rxjs": "^5.0.0-beta.12",
+    "@angular/common": "2.1.0",
```

Comment:
Hard coded deps in general because in our experience there used to be a lot of issues showing up just by updating to the wrong angular/angular-cli/webpack/typescript/etc version. Regarding typescript in particular, that is a dev dependency, so if I'm right it should not really affect library users. After all, typescript is the language that *happen to be* used to author the library, which is later consumed by users as plain old JS. 
Beside that, just make sure not to move to latest typescript (2.1.4), as that definitely caused some (known) issues with build, which in the end made us stick with a specific version.